subjects/rules.md

```diff
@@ -0,0 +1,50 @@
+# Common Validation rules
+
+## Validation request requirements
+
+In order to ease reviewing validation requests, ensure you match the following
+requirements before applying for a review.
+
+???+ success "Why the requirements?"
+
+    The goal is to ease reviewing and offer your the occasion
+    to conform to the Open Source standards such as documentation .
+
+The repository you submit **must** conform to the following.
+
+### Requirements
+
+Proper documentation **must** be easily accessible (i.e.: [README.md][github-readme],
+[mkdocs][mkdocs] or anything that is viewable and maintanable).
+
+???+ todo "Validation requests"
+    - program _should_ be able to execute the example(s)
+    - program _should_ accept input given in the subject description excepted
+    if adapting the format is relevant
+    - program _may_ output information in other formats than specified in the subject
+
+???+ todo "Linting"
+    - Repository **must** conform to at least one community linter ([pylint][pylint],
+    [tslint][tslint], `go fmt`).
```

Comment:
You could add [this link](https://blog.golang.org/gofmt) to `go fmt`, if you find it helpful.

Reply:
I added a link to go fmt to prevent any misunderstanding, thank ! :rocket: